### DISCUSSION TITLE
LeaderboardAPI.GetRangeRank 문제가 있는거 같네요

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#686] [5. Q&A]
월드 1.3.0 입니다 

리더보드 테스트 중인데요 LeaderboardAPI.GetRangeRank 가 문제가 있는거 같습니다 

문제는 다음과 같습니다 
시작 랭크~ 끝 랭크 까지 요청했을 경우 서로 다른 스코어를 가진 클라이언트 정보가 나와야 할것 같은데요 
동일한 클라이언트 이름이 나옵니다 

이때 rank, score는 문제 없이 출력 됩니다 
name만 문제가 있는것 같아요 확인 부탁 드려용 ~ 

![image](https://user-images.githubusercontent.com/50188795/166149120-cefaf664-2483-4926-b16b-02e745dbea45.png)
![image](https://user-images.githubusercontent.com/50188795/166149124-43953453-6700-4340-b6e4-578c1083d80a.png)

**Comment by smilejsu82** [comment]
name: ${result.rankInfo.myRank.name}`);
였네요... 

코드를 잘 안본 저의 불찰입니다 (__)

**Comment by 17beans** [comment]
덕분에 찾았습니다. 저도 같은 실수를 했네요 ㅎㅎ.. 예제 코드를 복사해서 사용했는데 이름 출력하는 부분만 myRank에서 가져오도록 되어있었네요 ㅋㅋㅋ
